lib/Service/RowService.php

```diff
@@ -183,7 +183,7 @@ public function create(?int $tableId, ?int $viewId, array $data):Row {
 	 * @throws InternalError
 	 */
 	private function cleanupData(array $data, array $columns, ?int $tableId, ?int $viewId): ?array {
```

Comment:
```suggestion
	private function cleanupData(array $data, array $columns, ?int $tableId, ?int $viewId): array {
```

Now that you always return an array, you probably can adust the return type. Also needs to be adjusted in the annotations then.